Tests/SentryTests/SentryScopeSwiftTests.swift

```diff
@@ -415,7 +415,55 @@ class SentryScopeSwiftTests: XCTestCase {
         // The number is kept small for the CI to not take too long.
         // If you really want to test this increase to 100_000 or so.
         testConcurrentModifications(asyncWorkItems: 2, writeLoopCount: 10, writeWork: { _ in
-            self.modifyScope(scope: scope)
```

Comment:
I moved the contents of the modifyScope function in here so the test is easier to read.